src/Tabs.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  tabs: Tab[],
+  onContent: (tabId:string) => void
+  selectedId: string | undefined
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, onContent, selectedId }) => {
+  const findSelectedTab = tabs.find(tab => tab.id === selectedId);
```

Comment:
You can transfer selectedTab from App component like props. You already have a selectedTab in state)

Reply:
No need to duplicate info sent in props. This is OK